plugins/modules/nd_setup.py

```diff
@@ -248,6 +255,31 @@
                 - The ASN of the BGP peer.
                 type: int
                 required: true
+  deployment_mode:
+    description:
+    - This is used for enabling the services between Orchestrator,
+      Fabric Controller and Insights during the initial installation.
+    - This option is only applicable for ND version 3.1.1 and later.
+    type: list
+    elements: str
+    choices: [ ndo, ndfc, ndi ]
+  external_services:
+    description:
+    - The persistent Service IPs/Pools to be provided.
+    - This can be used when I(deployment_mode) includes C(ndfc) or C(ndi) otherwise it will be ignored.
```

Comment:
```suggestion
    - This can be used when 0(deployment_mode=ndfc) or 0(deployment_mode=ndi) otherwise it will be ignored.
```

Reply:
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation

Reply:
```suggestion
    - This can be used when O(deployment_mode) includes V(ndfc) or V(ndi) otherwise it will be ignored.
```
Or since deployment_mode is a list:
```suggestion
    - This can be used when O(deployment_mode=[ndfc]) or O(deployment_mode=[ndi]) otherwise it will be ignored.
```
I found the latest more confusing because from a reader point of view, it seems like the only two options when to use external_services but you can also use it with a combination of ndi/ndo for example. what do you think

Reply:
I like the O V combination better